solutions/instances/variables.tf

```diff
@@ -439,6 +453,18 @@ variable "skip_cos_kms_auth_policy" {
   default     = false
 }
 
+variable "skip_cloud_logs_cos_auth_policy" {
+  type        = bool
+  description = "To skip creating an IAM authorization policy that allows the IBM Cloud logs to read the Cloud Object Storage instance, set this variable to `true`."
+  default     = false
+}
+
+variable "skip_at_cos_auth_policy" {
+  type        = bool
+  description = "To skip creating an IAM authorization policy that allows the Activity Traker to read from the Cloud Object Storage instance, set this variable to `true`."
```

Comment:
This isn't correct. The Activity Tracker service is getting object writer access to the COS instance